.github/workflows/nightly-ghcr.yaml

```diff
@@ -25,8 +25,11 @@ jobs:
       - name: Setup golang
         uses: ./.github/actions/golang
 
+      - name: Install UDS CLI
+        uses: ./.github/actions/install-uds-cli
+
```

Comment:
It might not matter but is there a reason for installing + using `uds run` for this first "bootstrap" vs using `maru`/`run` direct from maru upstream? `uds` is probably easier to install (brew) but felt odd at first glance that we would require `uds` to build `uds` 😆. This is probably a super edge case but I think you could end up in a weird situation if you needed a new maru feature to update a task, in a single PR - the `brew` installed uds-cli would have an older maru version.

Happy to punt on this one - just some food for thought.

Reply:
This one is intentional, we wanted to use `uds` to build `uds`. The edge case is a good point, my thinking was that we would want to know if there was a breaking Maru change so we could handle it on behalf of downstream teams

Reply:
That sounds good - I think this works either way to dogfood uds.